examples/react/zod/src/index.tsx

```diff
@@ -16,18 +16,28 @@ function FieldInfo({ field }: { field: FieldApi<any, any, any, any> }) {
   )
 }
 
+const formSchema = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+})
+
+type Form = z.infer<typeof formSchema>
```

Comment:
Renaming `formSchema` to e.g. `userSchema` and `Form` to `User` should make this example easier to follow. Otherwise the word "form" is used a lot…

Reply:
fixed :)